apps/astarte_realm_management/test/astarte_realm_management/queries_test.exs

```diff
@@ -304,75 +273,78 @@ defmodule Astarte.RealmManagement.QueriesTest do
 
     assert Queries.get_interfaces_list(realm_name) == {:ok, ["com.ispirata.Hemera.DeviceLog"]}
 
-    DatabaseQuery.call!(client, @insert_log_line0_device_a)
-    DatabaseQuery.call!(client, @insert_log_line1_device_a)
-    DatabaseQuery.call!(client, @insert_log_line0_device_b)
+    Repo.query!(@insert_log_line0_device_a)
+    Repo.query!(@insert_log_line1_device_a)
+    Repo.query!(@insert_log_line0_device_b)
 
     count =
-      DatabaseQuery.call!(client, @count_log_entries_for_device_a)
-      |> Enum.to_list()
+      Repo.query!(@count_log_entries_for_device_a)
+      |> Map.get(:rows, [[]])
+      |> hd()
+      |> hd()
 
-    assert count == [[count: 2]]
+    assert count == 2
 
     count =
-      DatabaseQuery.call!(client, @count_log_entries_for_device_b)
-      |> Enum.to_list()
+      Repo.query!(@count_log_entries_for_device_b)
+      |> Map.get(:rows, [[]])
+      |> hd()
+      |> hd()
 
-    assert count == [[count: 1]]
+    assert count == 1
 
     count =
-      DatabaseQuery.call!(client, @count_log_entries_for_device_c)
-      |> Enum.to_list()
+      Repo.query!(@count_log_entries_for_device_c)
+      |> Map.get(:rows, [[]])
+      |> hd()
+      |> hd()
 
-    assert count == [[count: 0]]
+    assert count == 0
 
     a_log_entry =
-      DatabaseQuery.call!(client, @a_log_entry_for_device_a)
-      |> Enum.to_list()
+      Repo.query!(@a_log_entry_for_device_a)
+      |> Map.get(:rows, [[]])
 
     assert a_log_entry == [
              [
-               device_id:
-                 <<83, 107, 226, 73, 170, 170, 78, 2, 149, 131, 90, 72, 51, 203, 254, 73>>,
-               path: "/",
-               reception_timestamp: 1_328_328_360_000,
-               reception_timestamp_submillis: 0,
-               v_applicationid: "this.is.a.bit.longer.string",
-               v_cmdline: "/usr/bin/things/test",
-               v_message: "testです",
-               v_monotonictimestamp: -1,
-               v_pid: -2,
-               v_timestamp: 1_328_241_960_000
+               <<83, 107, 226, 73, 170, 170, 78, 2, 149, 131, 90, 72, 51, 203, 254, 73>>,
+               "/",
+               ~U[2012-02-04 04:06:00.000Z],
+               0,
+               "this.is.a.bit.longer.string",
+               "/usr/bin/things/test",
+               "testです",
+               -1,
+               -2,
+               ~U[2012-02-03 04:06:00.000Z]
```

Comment:
Here we should also keep the name of the db column, as it is relevant to check whether object-aggregated values are correctly stored